docs/modules/ROOT/examples/management-center-ldap.yaml

```diff
@@ -7,12 +7,12 @@ spec:
   licenseKeySecretName: hazelcast-license-key
   securityProvider:
     ldap:
-      credentialsSecretName: ldap-credentianls
-      groupDN: ou=users,dc=example,dc=org
+      credentialsSecretName: ldap-credentials
+      groupDN: ou=groups,dc=example,dc=org
```

Comment:
Is there a specific reason for this change? If changed, then we must change the `userGroups` field accordingly